packages/web-components/src/components/rux-tabs/rux-tab-panels/rux-tab-panels.tsx

```diff
@@ -13,20 +13,20 @@ export class RuxTabPanels {
 
     connectedCallback() {
         this.el.setAttribute('style', 'position: relative; width: 100%;')
-    }
-
-    componentDidLoad() {
```

Comment:
This componentDidLoad was uneccesarry after my refactors. It was triggering re-runs of the slotted children method